### PR TITLE
Accounting Recipe

### PR DESCRIPTION
**Pull request by brittany-johnson**
TestyMcface: https://docs.google.com/spreadsheets/d/1QM28dLjuEH3xPtVty9l7Zfeqp4mg4fJtIAzv-oXH5Ik/edit#gid=2076382608

Trello Card: https://trello.com/c/5z2UfKZI/177-recipe-accounting

### CODE REVIEW

**Comment by helenemccarron** [comment]
The manifest also needs to be updated so the template dropdown will work properly.

File: books.txt [hunk 1]
```diff
@@ -28,6 +28,7 @@ declare -x -r RECIPE_DESC=(
```
Comment (helenemccarron) [inline books.txt]: The `RECIPE_DESC` needs to be updated as well for the dropdown to work.

**Comment by brittany-johnson** [comment]
@helenemccarron ready for review

File: books.txt [hunk 2]
```diff
@@ -2,7 +2,11 @@
 
 # These columns are  recipe_name and long description
 declare -x -r RECIPE_DESC=(
+<<<<<<< HEAD
```
Comment (helenemccarron) [inline books.txt]: mmmh I think the conflict wasn't resolved properly

File: books.txt [hunk 3]
```diff
@@ -28,6 +32,30 @@ declare -x -r RECIPE_DESC=(
 # - additionalStyleguideDirs : These specify which additional directories should be included when generating the styleguide
 
 BOOK_CONFIGS=(
+<<<<<<< HEAD
```
Comment (helenemccarron) [inline books.txt]: same here

File: recipes/output/manifest.json [hunk 4]
```diff
@@ -1,8 +1,14 @@
 [
 {
+<<<<<<< HEAD
```
Comment (helenemccarron) [inline recipes/output/manifest.json]: same here

**Pull request merged**